### DISCUSSION TITLE
T2dash or T2star for phantom generation

### DISCUSSION THREAD

**Discussion by h3lg3** [MRsources/MRzero-Core#88] [Q&A] [answered]
Hi! 
I want to generate MRzero phantoms based on actual qMRI measurements. I wanted to make sure that you actually use T2dash (T2') and **not** T2star (T2*) to specify the phantom. Could you please confirm that T2' is used for phantom specification? 

$\frac{1}{T2^{\prime}} = \frac{1}{T2^*} - \frac{1}{T2}$

Thanks a lot.
Helge

**Comment by mzaiss** [comment, marked as answer]
yes, t2' is defined.

be aware that the dB0 dependent part is not explicitly defined, but will
emerge implicitly, once you have higher phantom resolution than sequence
resolution

h3lg3 ***@***.***> schrieb am Mi., 27. Nov. 2024, 18:50:
